setup-ovs.sh

```diff
@@ -4,13 +4,29 @@ set -ex
 
 rm -rf /etc/NetworkManager/system-connections/*
```

Comment:
Not sure I want to, I would like to make sure that this folder is empty before we start.
This makes sure we are clean with no extra config files

Reply:
But isn't the logic of NMPATH means that only the path on `NMPATH` is relevant for NetworkManager?

Reply:
It is, but what if something will be added not by us and will effect the process?

Reply:
IIUC, NetworkManager should only care about the value of `NMPATH` 
If this assumption is correct, then if it is set to some other folder (like /etc/NetworkManager/system-connections-merged) then it will only look at that.

Reply:
Dont know, I mean this line was added by the CU in its own environment. 
I am think its not effecting our script its just a double verification that no older config files are alive.

Reply:
well if you think it's harmless then I won't insist :)

Reply:
I think so :)